lib/httplib/httpheaders.go

```diff
@@ -76,7 +118,11 @@ func getContentSecurityPolicyString(cspMaps ...map[string]string) string {
 
 	var cspStringBuilder strings.Builder
 	for _, k := range keys {
-		fmt.Fprintf(&cspStringBuilder, "%s %s; ", k, combined[k])
+		cspStringBuilder.WriteString(k)
+		for _, v := range combined[k] {
+			cspStringBuilder.WriteString(" " + v)
```

Comment:
```suggestion
			cspStringBuilder.WriteString(" ")
			cspStringBuilder.WriteString(v)
```

Avoids allocating a temporary string